scripts/experiment-loader.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Configuration for experimentation.
+ * @type {Object}
+ */
+const experimentationConfig = {
+  prodHost: 'www.my-site.com',
+  audiences: {
+    mobile: () => window.innerWidth < 600,
+    desktop: () => window.innerWidth >= 600,
+    // define your custom audiences here as needed
+  },
+};
```

Comment:
I'd move this back into `scripts.js`. This is project-specific config and code that we want them to manage.
The rest of the file is meant to be mostly static.